src/hintingtt.js

```diff
@@ -0,0 +1,3043 @@
+/* A TrueType font hinting interpreter.
+*
+* (c) 2017 Axel Kittenberger
+*
+* This file is covered by the MIT (expat) License,
+* see the LICENSE file in the root of this project.
```

Comment:
I agree it's not important. You can cut it if you want to. It's oft generally recommended a good idea for every file tough. src/opentype.js has for example.